README.md

```diff
@@ -9,7 +9,7 @@ Before using this library, run `main.py` in the `generator` directory with the p
 The CAN spec should be a YAML file with the following structure:
 ```yaml
 name: MYwhatever # Name of your CAN spec
-archs: # Allowed architectures for boards
+architectures: # Allowed architectures for boards
```

Comment:
I know that you mentioned that you made this change, but why did you want to change this name?

Reply:
More explicit.